Starwatch.Core/API/Rest/Route/AnnouncementRoute.cs

```diff
@@ -0,0 +1,387 @@
+﻿/*
+START LICENSE DISCLAIMER
+Starwatch is a Starbound Server manager with player management, crash recovery and a REST and websocket (live) API. 
+Copyright(C) 2022 Lachee
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see < https://www.gnu.org/licenses/ >.
+END LICENSE DISCLAIMER
+*/
+using System;
+using Starwatch.API.Rest.Routing;
+using Starwatch.Entities;
+using Starwatch.API.Rest.Route.Entities;
+using Starwatch.API.Rest.Serialization;
+using Starwatch.API.Web;
+using Starwatch.API.Gateway.Event;
+using Starwatch.API.Gateway;
+using System.Linq;
+using System.Threading.Tasks;
+using Starwatch.Monitoring;
+using Starwatch.Util;
+using System.Diagnostics;
+
+namespace Starwatch.API.Rest.Route
+{
+    [Route("/announcement", AuthLevel.Admin)]
+    class AnnouncementRoute : RestRoute, IGatewayRoute
```

Comment:
It doesn't seem that you use the gateway, does it require the `IGatewayRoute` then?

Reply:
Yeah you're right. I just copied from another route's code to get to there and left it in because I wasn't sure.